pkg/volume/util/fsquota/quota_linux.go

```diff
@@ -227,6 +227,9 @@ func clearQuotaOnDir(m mount.Interface, path string) error {
 	klog.V(4).Infof("clearQuotaOnDir %s", path)
 	supportsQuotas, err := SupportsQuotas(m, path)
 	if !supportsQuotas {
+		if err != nil {
```

Comment:
Should we be checking `if err != nil` before we attempt to check `supportsQuota`?

Reply:
yup, will address is similar to https://github.com/kubernetes/kubernetes/pull/87462#discussion_r370753862